fashion-example-query/tests/test_fashion_example_search.py

```diff
@@ -0,0 +1,182 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+
+import urllib.request
+import gzip
+import numpy as np
+
+from jina.flow import Flow
+from jina.logging.profile import ProgressBar
+from jina import Document
+from jina.executors.encoders import BaseImageEncoder
+
+from pkg_resources import resource_filename
+
+
+class MyEncoder(BaseImageEncoder):
+    """Simple Encoder used in :command:`jina hello-world`,
+        it transforms the original 784-dim vector into a 64-dim vector using
+        a random orthogonal matrix, which is stored and shared in index and query time"""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # generate a random orthogonal matrix
+        H = np.random.rand(784, 64)
+        u, s, vh = np.linalg.svd(H, full_matrices=False)
+        self.oth_mat = u @ vh
+        self.touch()
+
+    def encode(self, data: 'np.ndarray', *args, **kwargs):
+        # reduce dimension to 50 by random orthogonal projection
+        return (data.reshape([-1, 784]) / 255) @ self.oth_mat
+
+
+result = []
```

Comment:
please make this local to the `test` function

Reply:
Did you mean the 'result'?